lms/static/sass/_build-lms.scss

```diff
@@ -21,7 +21,7 @@
 @import 'shared/fields';
 @import 'shared/forms';
 @import 'shared/footer';
-@import 'shared/footer-edx'; // Replaces most of the footer partial. Will update footer to remove edx specific styles once feature flag removed.
+@import 'shared/footer-edx';
```

Comment:
I added another comment directly in `shared/footer` next to the styles we want to remove once V3 is turned on.